app/templates/mascots/commission.hbs

```diff
@@ -1 +1,130 @@
-{{outlet}}
\ No newline at end of file
+<section>
+
+  <h1>Mascot Commission Requests</h1>
+
+  <div class="left-col">
+    <p>Thanks so much for your interest in commissioning a Tomster or Zoey for your User Group (UG) or Company (Corp)! As per our {{link-to "FAQ" "mascots.faq"}}, assuming an approved use case, we'd love to work with you!</p>
+    <p>Most of the Tomsters and Zoeys we design fit into a somewhat standard mold. You can <a href="/pdfs/Mascot-Pricing.pdf">download the pricing sheet</a> to learn about the standard options.</p>
+    <p>Filling out this form is the first step to commissioning your Custom Ember Mascot. We'll get back to you as soon as we can with our thoughts and next steps.</p>
+
+    <form id="tomster-commission-form" name="tomster-commission-form" class="tomster-form wufoo topLabel page" accept-charset="UTF-8" autocomplete="off" enctype="multipart/form-data" method="post" novalidate action="https://tilde.wufoo.com/forms/q14mw9qi196f6oy/#public">
```

Comment:
I've just copied-and-pasted the existing form markup from the current site. I've made no attempt at rewriting or optimizing the HTML.